.vscode/settings.json

```diff
@@ -2,7 +2,7 @@
   "editor.insertSpaces": true,
   "editor.tabSize": 2,
   "editor.rulers" : [100],
-  "editor.acceptSuggestionOnEnter": "off",
+  "editor.acceptSuggestionOnEnter": "on",
```

Comment:
Are you sure you want to change this file?

Reply:
Surely not... 😄